widgets/BigQueryTable-batchsink.json

```diff
@@ -138,6 +138,17 @@
     {
       "label": "Advanced",
       "properties": [
+        {
+          "name": "jobLabels",
+          "label": "Job Label Key Value",
```

Comment:
Please also rename the label here to `BQ Job Labels` and update the docs.

Similar comment for BQ Multi Sink.

Reply:
> BQ Job Labels

Reply:
```json
"label": "BQ Job Labels"
```